testdata/project-v3-addon/PROJECT

```diff
@@ -3,13 +3,16 @@ layout: go.kubebuilder.io/v3-alpha
 projectName: project-v3-addon
 repo: sigs.k8s.io/kubebuilder/testdata/project-v3-addon
 resources:
-- group: crew
+- crdVersion: v1
+  group: crew
   kind: Captain
   version: v1
-- group: crew
+- crdVersion: v1
+  group: crew
   kind: FirstMate
   version: v1
-- group: crew
+- crdVersion: v1
+  group: crew
```

Comment:
It is missing the webhook version

Reply:
Webhooks aren't scaffolded by this testdata's generation script

Reply:
I checked that it has been stored now in : https://github.com/kubernetes-sigs/kubebuilder/pull/1644/files#diff-cd655d3efb3cc180279e89a469398876b374d03b66a1e65a1bef36f083ceb3c9R16 👍